ivy/functional/ivy/linear_algebra.py

```diff
@@ -337,19 +340,163 @@ def diagonal(
     >>> d = ivy.diagonal(x)
     >>> print(d)
     ivy.array([1., 4.])
+
     >>> d = ivy.diagonal(x, 1)
     >>> print(d)
     ivy.array([1.])
 
-    A 3-D Example
+    Functional Examples
```

Comment:
Hi @Sandstorm831 , thanks for making all the changes!

It seems you're still yet to move the examples above (Lines 338~346) down into `Functional Examples`. \
Please move these 2 examples into `Function Examples` and then delete the heading `Examples`.

Once you make this change I'll merge in all your edits.

Reply:
Hello @chie2727 , Thanks for giving the final change prompts, I have implemented all the required changes, please look through them once.